packages/table/test/isotest.js

```diff
@@ -20,10 +20,23 @@ const customProps = {
     },
 };
 
+const classNameChildList = [
+    "CopyCellsMenuItem"
+]
+
+const skipList = [
```

Comment:
@giladgray Classname doesn't propagate for these three components, but i'm not sure of the best way to handle these. It looks like they are just wrappers for eventHandlers, in which case maybe className doesn't need to propagate?

Reply:
`ResizeHandle` - add support

the other two, you're right, they're the exceptions!